README.md

```diff
@@ -22,3 +22,155 @@ e.g.
 ```
 npm i @fastify/cors -w @mykomap/back-end --save
 ```
+
+## Installation and updates
+
+Conceptually, installation of these applications require:
+- deploying the front-end as content to be served on the web
+  - configure it to use the correct path for the back end (typically `/api`)
+  - also configure API keys for GlitchTip and MapTiler services
+- deploying the back end to be run as a persistent node process
+  - configure it to use the right path to the data folder
+- deploy the data in the path expected by the back-end
+- reverse-proxying the back end to be accessible on the same domain as the front end
+
+### Deploying on a DCC server, manually
+
+DCC deployment uses a Linux server running Apache, an appropriate
+installation of NodeJS, with SystemD that supports user-mode
```

Comment:
We should maybe mention that NodeJS installation is controlled with ASDF (and the version defined in source code)?